src/OpenFeature.Contrib.ConfigCat/UserBuilder.cs

```diff
@@ -0,0 +1,51 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using ConfigCat.Client;
+using OpenFeature.Model;
+
+namespace OpenFeature.Contrib.ConfigCat
+{
+    internal static class UserBuilder
+    {
+        private static readonly string[] PossibleUserIds = { "ID", "IDENTIFIER" };
+
+        internal static User BuildUser(this EvaluationContext context)
+        {
+            if (context == null)
+            {
+                return null;
+            }
+
+            var user = context.TryGetValuesInsensitive(PossibleUserIds, out var pair)
+                ? new User(pair.Value.AsString)
+                : new User(Guid.NewGuid().ToString());
+
+            foreach (var value in context)
+            {
+                switch (value.Key.ToUpperInvariant())
+                {
+                    case "EMAIL":
+                        user.Email = value.Value.AsString;
+                        continue;
+                    case "COUNTRY":
+                        user.Country = value.Value.AsString;
+                        continue;
+                    default:
+                        user.Custom.Add(value.Key, value.Value.AsString);
+                        continue;
+                }
```

Comment:
Same as https://github.com/open-feature/dotnet-sdk-contrib/pull/119/files#r1438292902

Reply:
Added a README with that info.